### PR TITLE
Add myTrans options when upload translation document

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1184299

See also:
API: https://github.com/zanata/zanata-api/pull/24
Client: https://github.com/zanata/zanata-client/pull/46

### CODE REVIEW

File: zanata-war/src/main/java/org/zanata/service/impl/TranslationServiceImpl.java [hunk 1]
```diff
@@ -825,21 +829,13 @@ public String apply(TextFlowTarget input) {
...
-                                personDAO.makePersistent(hPerson);
-                            }
+                        if(assignCreditToUploader){
+                            HPerson hPerson = authenticatedAccount.getPerson();
                             hTarget.setTranslator(hPerson);
                             hTarget.setLastModifiedBy(hPerson);
                             actorId = hPerson.getId();
-                        } else {
+                        }
+                        else {
```
Comment (huangp) [inline zanata-war/src/main/java/org/zanata/service/impl/TranslationServiceImpl.java]: I think you will fail checkstyle here

**Comment by huangp** [comment]
:+1:

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested (via check on database update - no ui related changes).

**Comment by aeng** [comment]
@djansen-redhat, thats for https://bugzilla.redhat.com/show_bug.cgi?id=1188077.

**Comment by djansen-redhat** [comment]
@aeng :facepunch: 
Looks fine to me.

**Comment by aeng** [comment]
@zanata-jenkins retest this please

**Comment by aeng** [comment]
@djansen-redhat, nothing changed except for merging from integration/master since @huangp reviewed the pull request.

**Comment by djansen-redhat** [comment]
@aeng That "fix tests" does a bit more than fix tests :wink: 
I've [done my part](http://i.imgur.com/YC2K5ap.png), if you want to label it reviewed and proceed, you have QE clearance.

**Comment by aeng** [comment]
The "fix tests" commit is really just fixing backwards compatibility test. If you mean the download and upload dialog, its implemented before the review.

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/f374e740fe602175e5557460527271ed54b2fc3e

**Pull request closed without merging**